src/components/main-page/main-page.jsx

```diff
@@ -5,23 +5,25 @@ import Header from '../header/header';
 import CardList from '../card-list/card-list';
 import Map from '../map/map';
 import {OfferType} from '../../typings/offer';
-import {CITY_CORDS} from '../../const';
+import {connect} from 'react-redux';
+import {ActionCreator} from '../../store/action';
+import {CITIES} from '../../const';
 
 const MainPage = (props) => {
-  const {offers} = props;
+  const {offers, city, onCityChange} = props;
 
   return (
     <React.Fragment>
       <div className="page page--gray page--main">
         <Header />
         <main className="page__main page__main--index">
           <h1 className="visually-hidden">Cities</h1>
-          <FilterBar/>
+          <FilterBar onCityChange={onCityChange} currentCity={city}/>
```

Comment:
Давай тут немного больше кода напишем, чтобы критерию [Д2](https://up.htmlacademy.ru/react/6/criteries#d2) соответствовать:
```suggestion
          <FilterBar onCityChange={handleCityChange} currentCity={city}/>
```
А выше в компоненте объявим эту функцию:
```js
const handleCityChange = (city) => onCityChange(city)
```